Paws4Me/Paws4Me/Data/LocalPetNameViewController.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  LocalPetNameViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/10.
+//
+
+import CoreData
+import UIKit
+
+class LocalPetViewController: UIViewController, UITableViewDelegate {
```

Comment:
Move `protocol` conformance to separate `extension` 😄

Reply:
Will do so. I keep forgetting about this!